src/presentation/controller/registration/Controller.ts

```diff
@@ -70,16 +84,22 @@ class RegistrationController extends AbstractController {
           result.submissionId
         );
 
+        await this.cacheService.removeDataFromCache(
+          session,
+          `registration_${RegistrationPageType.whichType}`
```

Comment:
`registration_` appears here and then also a couple of times in the controller code, lower down.

Might be worth defining a constant in `constants.ts`, something like `CACHE_KEY_PREFIX_REGISTRATION = "registration_"` to avoid the repetition.

Up to you.

Reply:
constant added